src/status_im/contexts/wallet/send/from/view.cljs

```diff
@@ -23,10 +24,17 @@
 
 (defn- render-fn
   [item]
-  (let [network-details (rf/sub [:wallet/network-details])]
+  (let [network-details     (rf/sub [:wallet/network-details])
```

Comment:
This is something we can avoid. The subscription can be moved to the parent component instead of being tracked by re-frame on every item.

Reply:
Done